test/helm-test-e2e.sh

```diff
@@ -20,6 +20,10 @@ kubectl -n kube-system create sa tiller
 kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
 linux-amd64/helm init --service-account tiller --upgrade
 
+linux-amd64/helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+linux-amd64/helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com/
+linux-amd64/helm repo add kubernetes-charts http://storage.googleapis.com/kubernetes-charts
```

Comment:
is this necessary? If charts are using this URL they should be updated to the correct one.